insights/parsers/brctl_show.py

```diff
@@ -102,6 +98,8 @@ def parse_content(self, content):
                 else:
                     iface_lst.append(line.strip())
                     br_mapping[iface] = iface_lst
+        else:
+            raise ParseException("Invalid Data Found")
 
         for entry in self.data:
```

Comment:
Better shift this `for` into the `if iface` clause.
and remove the last `return`.